dev-java/core-specs-alpha/core-specs-alpha-0.2.56.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit java-pkg-2 java-ant-2
+
+EGIT_REF="053d789"
+
+MY_PN=${PN//-/.}
+MY_P=${MY_PN}-${PV}
+
+DESCRIPTION="A Clojure library with specs to describe Clojure core macros and functions."
+HOMEPAGE="https://clojure.org/ https://github.com/clojure/core.specs.alpha"
+SRC_URI="https://github.com/clojure/${MY_PN}/archive/${MY_P}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="EPL-1.0"
+SLOT="0.2"
+KEYWORDS="~amd64 ~x86 ~x86-linux"
+
+CDEPEND="dev-java/ant-core:0"
+RDEPEND=">=virtual/jre-1.8:*"
+DEPEND="
+	${CDEPEND}
+	>=virtual/jdk-1.8:*"
```

Comment:
It doesn't look like you need `CDEPEND` here at all.

Reply:
Moved directly on DEPEND and removed CDEPEND at all